### ISSUE TITLE
Collapse widget prevents clicks on inner radio buttons

### ISSUE THREAD

**Issue by samuel9554** [twbs/bootstrap#19938]
I have a issue with bootstrap. When the div is extend the radio button will block and won't work.

Here's the code :

```
<form name="today_task" action="" method="post">
    <div data-toggle="collapse" id="add_task" class="collapse">
        <textarea name="task_content" style="width: 100%;" placeholder="Enter task"></textarea>
        <input type="radio" name="priority" value="3"> High Priority <br />
        <input type="radio" name="priority" value="2" checked> Medium Priority <br />
        <input type="radio" name="priority" value="1"> Low Priority <br />
    </div>
    <a class="accordion-toggle" data-toggle="collapse" data-target="#add_task" href="#add_task"><input type="button" name="submit_new_task" id="add_task_button" value="Add Task" class="btn btn-success" style="width: 100%;" onClick="submit_button()"></a>                                           
</form>
```
Here's the problem :https://jsfiddle.net/g14ajcgh/

**Comment by twbs-lmvtfy** [comment]
Hi @samuel9554!

You appear to have posted a live example (https://fiddle.jshell.net/g14ajcgh/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* JSFiddle inserts the contents of its "HTML" pane within ```` <body>...</body> ```` tags, so you cannot include your own ```` <body> ```` tag in your JSFiddle.
* line 80, column 1 thru column 6: Start tag ```` body ```` seen but an element of the same type was already open.
* line 82, column 3 thru column 50: Bad value ````  ```` for attribute ```` action ```` on element ```` form ````: Must be non-empty.
* line 89, column 127 thru column 279: The element ```` input ```` must not appear as a descendant of the ```` a ```` element.
* line 95, column 1 thru column 7: Saw an end tag after ```` body ```` had been closed.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
X-Ref: #14991

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
